From bb61f0e4ade32b11f223005ed350844e53f74e81 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Thu, 21 Jul 2016 11:34:48 -0700
Subject: [PATCH 1800/2748] MAPREDUCE-6738. TestJobListCache.testAddExisting
 failed intermittently in slow VM testbed.
 Contributed by Junping Du.

(cherry picked from commit ecff7d03595b45d7a5d6aed73b9931fef85a56ee)

Change-Id: I3a6d2d66c1ce9d7d4ca38830a333ed5f97de9cb1
---
 .../hadoop/mapreduce/v2/hs/TestJobListCache.java   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/TestJobListCache.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/TestJobListCache.java
index 6ebbb7c..ba5599c 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/TestJobListCache.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/src/test/java/org/apache/hadoop/mapreduce/v2/hs/TestJobListCache.java
@@ -31,7 +31,7 @@
 
 public class TestJobListCache {
 
-  @Test (timeout = 1000)
+  @Test (timeout = 5000)
   public void testAddExisting() {
     JobListCache cache = new JobListCache(2, 1000);
 
-- 
1.7.9.5

